devel/ci/integration/bodhi/Dockerfile-rawhide

```diff
@@ -7,6 +7,12 @@ LABEL \
 
 # For integration testing we're using the infrastructure repo
 RUN curl -o /etc/yum.repos.d/infra-tags.repo https://infrastructure.fedoraproject.org/cgit/ansible.git/plain/files/common/fedora-infra-tags.repo
+# work around https://bugzilla.redhat.com/show_bug.cgi?id=1699396
+RUN rpm --import "$(awk -F= '{if ($0 ~ "gpgkey *=") {print $2;}}' < /etc/yum.repos.d/infra-tags.repo)"
+# deal with https://fedoraproject.org/wiki/Changes/Set_skip_if_unavailable_default_to_false
+RUN dnf install -y dnf-plugins-core
+RUN dnf config-manager infrastructure-tags --setopt skip_if_unavailable=True --save
```

Comment:
Every command adds a layer in the container, do you know if Randy and Aurélien preferred the one command / line approach? Otherwise, it is often recommended, to have multiple command per "RUN" using ``\`` to delimit them.

Reply:
I don't know their preference. In the Dockerfiles there are instances of one RUN containing multiple commands (though not many), but also multiple RUN lines each containing a single command.